qemu/tests/cfg/live_snapshot_chain.cfg

```diff
@@ -14,8 +14,6 @@
     check_alive_cmd = "ls"
     kill_vm = yes
     remove_snapshot_images = yes
-    pre_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
-    post_command = "sync && echo 3 > /proc/sys/vm/drop_caches"
```

Comment:
`drop_caches` should __ONLY__ be performed while testing disk performance, otherwise it's linux/kernel purpose to maintain the caches. So if following tests are failing it's either a kernel bug (which should not be hidden) or test-case bug (as seen on cgroup tests, because it uses `MemFree` instead of `MemAvailable`) which again means one should not hide it, but identify and fix it.

Reply:
@ldoktor statement is correct, but I can understand the motivation to keep this (due to the type of testing @luckyh team performs).

@luckyh does the downstream configuration repo allow for configuration also at the test level?  If so, it would be the perfect place to put this.

Reply:
@clebergnu yes, it does, thank you for pointing this out.

Reply:
@luckyh do I understand your message correctly that it's fine to include this commit now and you'll deal with this downstream? (if the problem arises, I mean this workaround was introduced long time ago so maybe it'll work seamlessly without it...)

Reply:
@ldoktor right, I'll add them into downstream configuration. (I hope it could work without the workaround too, but I have no knowledge of that so far)